libs/shared/l10n/src/lib/localizer/localizer.base.ts

```diff
@@ -0,0 +1,68 @@
+import { FluentBundle, FluentResource } from '@fluent/bundle';
+import type { ILocalizerBindings } from './localizer.interfaces';
+
+export class LocalizerBase {
```

Comment:
As a follow up task, the idea is to update `fxa-auth-server/lib/l10n/index.ts > Localizer` to inherit from `LocalizerBase`, hence why it's using `ILocalizerBindings` etc.